drivers/media/platform/nuvoton/npcm750_vcd.c

```diff
@@ -133,6 +138,64 @@
 #define  VCD_INTE_BSD3_IE	BIT(13)
 #define  VCD_INTE_VAL	(VCD_INTE_DONE_IE | VCD_INTE_IFOR_IE)
 
+#define VCD_RCHG	0x8028
+#define VCD_RCHG_TIM_PRSCL  GENMASK(12, 9)
+#define VCD_RCHG_IG_CHG2  GENMASK(8, 6)
+#define VCD_RCHG_IG_CHG1  GENMASK(5, 3)
+#define VCD_RCHG_IG_CHG0  GENMASK(2, 0)
+#define VCD_RCHG_TIM_PRSCL_OFFSET 9
```

Comment:
This is minor:
I don't see you are using those _OFFSET registers, but if you do want the put them before and then use them:
#define VCD_RCHG_TIM_PRSCL_OFFSET 9
#define VCD_RCHG_IG_CHG2_OFFSET 6
#define VCD_RCHG_IG_CHG1_OFFSET 3
#define VCD_RCHG_IG_CHG0_OFFSET 0
#define VCD_RCHG_TIM_PRSCL GENMASK(12, VCD_RCHG_TIM_PRSCL_OFFSET)
#define VCD_RCHG_IG_CHG2 GENMASK(8, VCD_RCHG_IG_CHG2_OFFSET)
#define VCD_RCHG_IG_CHG1 GENMASK(5, VCD_RCHG_IG_CHG1_OFFSET)
#define VCD_RCHG_IG_CHG0 GENMASK(2, VCD_RCHG_IG_CHG0_OFFSET)